content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,202 @@ title: Instrumentation
 weight: 3
 ---
 
-This guide will cover creating and annotating spans, creating and annotating metrics, how to pass context, and a guide to automatic instrumentation for JavaScript. This simple example works in the browser as well as with Node.js
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+If a `TracerProvider` is not created, either explicitly or via SDK APIs, then
+the OpenTelemetry APIs for tracing will use a no-op implementation and fail to
+generate data.
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+### Node.js
 
-function doWork() {
-  console.log('work...');
-  // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
-}
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-node
 ```
 
-## Initializing a Tracer
+Next, create a separate `tracing.js` file that has all the SDK initialization
+code in it:
 
-As you have learned in the previous [Getting Started](../getting-started) guide you
-need a TracerProvider and an Exporter. Install the dependencies and add them to
-the head of your application code to get started:
+```js
+import opentelemetry from "@opentelemetry/api";
+import nodeOTel from "@opentelemetry/sdk-node";
+import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-base';
 
-```shell
-npm install @opentelemetry/api
-npm install @opentelemetry/sdk-trace-base
+const exporter = new ConsoleSpanExporter();
+
+const sdk = new nodeOTel.NodeSDK({
+  traceExporter: exporter,
+});
+
+sdk.start()
```

Comment:
important to know that as of right now the `sdk.start()` method is async and it is possible that if a span is created in the next milliseconds it may not be initialized early enough to catch it.